test/e2e/service.go

```diff
@@ -45,6 +46,17 @@ var _ = Describe("Services", func() {
 		Expect(err).NotTo(HaveOccurred())
 		namespace0 = "e2e-ns-" + dateStamp() + "-0"
```

Comment:
Yes, there is a test that uses two different namespaces. They are time-stamped, so it's fine to leave them as they are.